app/src/main/java/com/example/tsundokun/ui/home/HomeScreen.kt

```diff
@@ -1,17 +1,418 @@
+                    modifier = Modifier
+                        .padding(horizontal = 8.dp)
+                        .height(96.dp)
+                        .weight(3f),
+                )
+            }
+            Row(verticalAlignment = Alignment.CenterVertically) {
+                Image(
+                    painter = painterResource(R.drawable.ic_qiita),
+                    contentDescription = "qiita",
+                    modifier = Modifier
+                        .padding(horizontal = 8.dp)
+                        .clip(RoundedCornerShape(50))
+                        .weight(1f),
+                )
+                Text(
+                    text = LocalContext.current.getString(R.string.sample_web_page_type),
+                    modifier = Modifier
+                        .padding(8.dp)
+                        .weight(3f),
+                    style = MaterialTheme.typography.bodyMedium,
+                )
+                Spacer(modifier = modifier.weight(2f))
+                IconButton(onClick = { /* TODO */ }) {
+                    Icon(
+                        imageVector = Filled.FavoriteBorder,
+                        contentDescription = stringResource(R.string.button_favorite_description),
+                        modifier = modifier
+                            .weight(1f)
+                            .width(20.dp),
+                    )
+                }
+                IconButton(onClick = { /* TODO */ }) {
+                    Icon(
+                        imageVector = Filled.Share,
+                        contentDescription = stringResource(R.string.button_share_description),
+                        modifier = modifier
+                            .weight(1f)
+                            .width(20.dp),
+                    )
+                }
+                IconButton(onClick = { /* TODO */ }) {
+                    Icon(
+                        imageVector = Filled.MoreVert,
+                        contentDescription = stringResource(R.string.button_morevert_description),
+                        modifier = modifier
+                            .weight(1f)
+                            .width(20.dp),
+                    )
+                }
+            }
+        }
+    }
+}
+
+/*
+ * リストの各要素であるカードのプレビュー
+ */
+@Preview
+@Composable
+private fun WebPageCardPreview() {
+    WebPageCard(WebPage(R.string.sample_web_page_title, R.drawable.sample_image))
+}
+
+/*
+ * リスト全体のプレビュー
+ */
+@Preview(showBackground = true)
+@Composable
+fun WebPagePreview() {
+    TsundokunTheme {
+        WebPageListScreen()
+    }
+}
+
+/*
+ * FAB(追加ボタン)
+ */
+@Composable
+fun AddFab() {
+    ExtendedFloatingActionButton(
+        text = { Text(text = stringResource(R.string.fab_add)) },
+        icon = { Icon(Filled.Add, contentDescription = stringResource(R.string.fab_add)) },
+        onClick = { /*TODO*/ },
+    )
+}
+
+/*
+ * FAB(追加ボタン)のプレビュー
+ */
+@Preview
+@Composable
+private fun AddFabPreview() {
+    AddFab()
```

Comment:
他のPreview Composableと同じように、`TsundokunTheme` でwrapすると良さそうです
```suggestion
TsundokunTheme {
    AddFab()
}
```